src/perfect/index.js

```diff
@@ -0,0 +1,19 @@
+/* Perfect Number
+ * Function: isPerfect()
+ */
+
+const validate = require('../validation/positive-integer');
+
+exports.isPerfect = num => {
+    validate(num, 'isPerfect');
+    var count  = 0;
+    for(var i=1; i<num; i++)
+    {
+        if(num%i==0)
```

Comment:
use **===** instead of == in line 12
```suggestion
        if(num%i===0)
```